pom.xml

```diff
@@ -6,7 +6,7 @@
 	<groupId>SynapseAWSConsoleLogin</groupId>
 	<artifactId>SynapseAWSConsoleLogin</artifactId>
 	<packaging>war</packaging>
-	<version>1.0</version>
+	<version>${git.commit.time}-${git.commit.id.abbrev}</version>
```

Comment:
remember, these need to match up https://github.com/Sage-Bionetworks/synapse-login-scipool/blob/develop/.elasticbeanstalk/config.yml#L10

Reply:
This is overridden in pom.xml by <finalName>. I removed the version altogether there.

Reply:
when built I see `target/SynapseAWSConsoleLogin.war` while the config.yaml references  `target/SynapseAWSConsoleLogin-1.0.war`

```
deploy:
  artifact: target/SynapseAWSConsoleLogin-1.0.war
```

They need to match.

Reply:
I was gonna change the config.yaml. Unless you prefer I change the pom.xml <version> back to '1.0' and remove the override. That way the artifact is named after major version.

Reply:
It doesn't matter, whichever way you want to do it is fine.  It just needs to match up